model/task/task.go

```diff
@@ -1419,6 +1425,29 @@ func DeactivateStepbackTasksForProject(projectId, caller string) error {
 	return nil
 }
 
+// DeactivateStepbackTaskForProject deactivates and aborts the matching stepback task.
+// Will be used instead of LegacyDeactivateStepbackTasksForProject as part of EVG-17947
+func DeactivateStepbackTaskForProject(projectId, buildVariantName, taskName, caller string) error {
```

Comment:
Since this is no longer for an entire project, can we call this something else, like `DeactivateStepbackTask`?